cylc/flow/remote.py

```diff
@@ -195,16 +195,17 @@ def construct_rsync_over_ssh_cmd(
     """
     dst_host = get_host_from_platform(platform, bad_hosts=bad_hosts)
     ssh_cmd = platform['ssh command']
-    rsync_cmd = [
-        "rsync",
+    command = platform.get('rsync command', 'rsync')
```

Comment:
If you've defined the default in the cfgspec it doesn't need to be hardcoded here.

```suggestion
    command = platform['rsync command']
```